ext/sqlite3/sqlite3.stub.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+class SQLite3
+{
+    function __construct(string $filename, int $flags = SQLITE3_OPEN_READWRITE | SQLITE3_OPEN_CREATE, ?string $encryption_key = null);
+
+    function open(string $filename, int $flags = SQLITE3_OPEN_READWRITE | SQLITE3_OPEN_CREATE, ?string $encryption_key = null): void;
```

Comment:
Unfortunately, adding a return type to non-final classes is a BC break (it will be enforced in inheritance). For method return values we should always use `@return` for now. We should discuss on internals whether we want to actually enforce those return types in PHP 8 (which is definitely a possibility).